x-pack/plugins/stack_alerts/server/rule_types/es_query/rule_type.ts

```diff
@@ -135,6 +141,20 @@ export function getRuleType(
     }
   );
 
+  const alerts: IRuleTypeAlerts<RuleAlertData> = {
+    context: 'stack.esquery',
```

Comment:
This will write out all alerts into an `.alerts-stack.esquery-default` index. Are we good with this @kobelb @mikecote? What is the plan when querying the alerts indices to populate the alerts table?

Reply:
Writing these alerts to `.alerts-stack.esquery-default` is 👍 ; however, we'll need to verify that we can query the alerts properly. @XavierM was going to verify whether or not the changes we're making in https://github.com/elastic/kibana/pull/162605 are sufficient to fix this. At a minimum, we should not merge this PR before kibana#162605.

Reply:
Have we put some thoughts on whether we want esquery to be its own index or to have it shared with index threshold and other stack rule types?

Reply:
I compared EsQuery, IndexThreshold and GeoContainment rule types.
EsQuery and IndexThreshold somehow similar but GeoContainment has completely different fields.
Even if we wanted to share the index, it would be between EsQuery and IndexThreshold.